build.sbt

```diff
@@ -2,5 +2,7 @@ name := "aoc2020"
 
 version := "0.1"
 
-scalaVersion := "2.13.4"
+scalaVersion := "2.13.3"
```

Comment:
Das musste ich down-graden damit mein LSP server/metals funktioniert,

Reply:
The documentation of metals states that Scala 2.13.4 is compatible (https://scalameta.org/metals/docs/editors/vscode.html#requirements).
But not special 2.13.4 feature is needed for this project :)